cylc/flow/cfgspec/workflow.py

```diff
@@ -1715,10 +1715,12 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     Custom outputs must satisfy these rules:
 
                     .. autoclass:: cylc.flow.unicode_rules.TaskOutputValidator
+                       :no-index:
```

Comment:
I don't think it works with the hyphen.

The [documentation](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoclass) is confusing IMO - it looks like it should be `:no-index:` but a search for [`:noindex:`](https://www.sphinx-doc.org/en/master/search.html?q=%3Anoindex%3A) suggests otherwise.

Reply:
Ok, changed to `:noindex` and it still works on 7.2